### DISCUSSION TITLE
File transcription language en-GB

### DISCUSSION THREAD

**Discussion by bowen-facere** [deepgram/community#884] [General help] [answered]
Last week when we were testing transcription for a file using the language as `en-GB` we were seemingly getting correct British spelling back from the the API. However testing again today it looks like all words are being returned as American english.

We are using the python sdk and constructing the `PrerecordedOptions` like so:
```
options = PrerecordedOptions(
                model="nova-2",
                smart_format=True,
                language="en-GB",
                punctuate=True,
                keywords=prompts.split(" ") if prompts else None,
            )
```
Is anyone else experiencing this this issue? 

I also tried to see if this was replicable using the playground and it appears to be the case.

https://playground.deepgram.com/?endpoint=listen&smart_format=true&language=en-GB&model=nova
With the following audio file: https://github.com/user-attachments/assets/eba20159-758b-4979-9b7c-668423658723

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @bowen-facere, unfortunately while our `en-GB` model is trained on audio data including British English accents, and can therefore transcribe British English with high accuracy, all of our data labeling standards apply US English, so transcripts in any English dialect use American spellings. 

See another report of this current state: https://github.com/orgs/deepgram/discussions/749

If you require British spellings, your best options may be either to perform your own text cleaning (for instance, check out [british-american-words.txt](https://gist.github.com/ZekNikZ/5e7dd531df99be4408bd768ded36aad9), [convert-british-to-american-spellings](https://github.com/HoldOffHunger/convert-british-to-american-spellings), [American-British-English-Translator](https://github.com/hyperreality/American-British-English-Translator)), or to perhaps leverage an LLM that could convert from US to UK spellings.

**Reply by bowen-facere** [reply]
Hi @jkroll-deepgram, really appreciate the response. It seems like there are cases where the spelling does come out transcribed as British english, is this expected?

**Reply by jkroll-deepgram** [reply]
Hi @bowen-facere, I would be curious to see examples of British spellings in transcripts, if you can provide Deepgram request IDs, or even just excerpts of transcripts.

The foundation of which spellings the model outputs is how we label the input annotated audio for model training. Our style guide is purely US English. However, perhaps there are cases where the model has seen a word in one context, and applies it to another. 

For instance, once in the past I was trying to get our model to transcribe "staying mum" about something, and I was seeing that it sometimes got "staying mum", but also often outputted "staying mom". In another context with a British speaker, I could then see the model being able to correctly transcribe "mum" in British English.

On the other hand, I would be surprised if the model ever uses purely British spellings, like "colour", "analyse", etc, since there's no case in which that spelling would be used in US English. (Now if you do have examples like that, I would be happy to share with our data team and see what hypotheses they have about this occurrence!)

**Reply by bowen-facere** [reply]
Hi @jkroll-deepgram, unfortunately I'm unable to find the exact request-id, however I do have a screenshot of this output:

<img width="787" alt="image" src="https://github.com/user-attachments/assets/a0e4951c-b00c-4672-8300-a82b0242716c">

I believe this would've been from 1 Aug UTC, unfortunately I'm not sure the exact time.

**Reply by jkroll-deepgram** [reply]
Hi @bowen-facere, it looks like you have around 100 requests from that day, so I won't jump into looking through those right now :) 

That transcript is interesting, though. I wonder if that last line with the list of British spellings is from some other source and it was concatenated into the Deepgram transcript? It looks out of place with the sentence formatting that Deepgram uses for smart-formatted requests.

**Reply by bowen-facere** [reply]
Hi @jkroll-deepgram, I did look into that possibility, however the codebase we used to generate this transcript was not setup to invoke any asr resource other than deepgram. The API-key that was used for that request is the one named `joji-key` on our account, if that helps narrow it down.

**Reply by bowen-facere** [reply]
Hi @jkroll-deepgram here is a request id `9de71640-a1ee-4036-a1a4-9c3a2625b4d9` where we notice that "Dialog" and "Dialogue" appear.

**Reply by jkroll-deepgram** [reply, marked as answer]
Hi @bowen-facere, thanks for tracking down the request ID and sharing this example! I was able to replicate it.

I checked with our data labeling team, and they explained that this is a case where both spellings are valid in US English ([reference](https://writingexplained.org/dialog-vs-dialogue-difference)), so our training data wasn't normalized to a single spelling.

This does illustrate that there can still be some edge cases and variations, so it's not impossible for British English spellings to be used in limited cases. But by and large, all of our English transcripts will use US English spellings.

**Reply by bowen-facere** [reply]
Hi @jkroll-deepgram really appreciate the investigation, thank you.
